mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/mapping/definition/annotation/processing/impl/AbstractFieldAnnotationProcessor.java

```diff
@@ -42,12 +43,20 @@ public final void process(PropertyMappingStep mappingContext, A annotation,
 		PropertyMappingFieldOptionsStep<?> fieldContext =
 				initFieldMappingContext( mappingContext, annotation, cleanedUpRelativeFieldName );
 
+		ValueBinderRef valueBinder = getValueBinder( annotation );
 		ValueBinder binder = createValueBinder(
 				getValueBridge( annotation ),
-				getValueBinder( annotation ),
+				valueBinder,
 				context
 		);
-		fieldContext.valueBinder( binder );
+
+		if ( valueBinder.params() != null ) {
+			Map<String, Object> params = ParameterAnnotationsReader.toMap( valueBinder.params() );
```

Comment:
What's the point of exposing a method in the `context` if you don't use it? :)

```suggestion
			Map<String, Object> params = context.toMap( valueBinder.params() );
```